apps/web/components/v2/apps/CalendarListContainer.tsx

```diff
@@ -0,0 +1,325 @@
+                  )}
+                  onChanged={() => props.onChanged()}
+                />
+              }
+            />
+          ))}
+        </List>
+      )}
+    />
+  );
+}
+
+function ConnectedCalendarsList(props: Props) {
+  const { t } = useLocale();
+  const query = trpc.useQuery(["viewer.connectedCalendars"], { suspense: true });
+  const { fromOnboarding } = props;
+  return (
+    <QueryCell
+      query={query}
+      empty={() => null}
+      success={({ data }) => {
+        if (!data.connectedCalendars.length) {
+          return null;
+        }
+        return (
+          <List className="flex flex-col gap-6">
+            {data.connectedCalendars.map((item) => (
+              <Fragment key={item.credentialId}>
+                {item.calendars ? (
+                  <IntegrationListItem
+                    slug={item.integration.slug}
+                    title={item.integration.title}
+                    logo={item.integration.logo}
+                    description={item.primary?.externalId || "No external Id"}
+                    actions={
+                      <div className="w-32">
+                        <DisconnectIntegration
+                          credentialId={item.credentialId}
+                          trashIcon
+                          onSuccess={props.onChanged}
+                          buttonProps={{ className: "border border-gray-300" }}
+                        />
+                      </div>
+                    }>
+                    {!fromOnboarding && (
+                      <>
+                        <p className="px-4 pt-4 text-sm text-neutral-500">{t("toggle_calendars_conflict")}</p>
+                        <ul className="space-y-2 p-4">
+                          {item.calendars.map((cal) => (
+                            <CalendarSwitch
+                              key={cal.externalId}
+                              externalId={cal.externalId}
+                              title={cal.name || "Nameless calendar"}
+                              type={item.integration.type}
+                              defaultSelected={cal.isSelected}
+                              destination={cal.externalId === props.destinationCalendarId}
+                            />
+                          ))}
+                        </ul>
+                      </>
+                    )}
+                  </IntegrationListItem>
+                ) : (
+                  <Alert
+                    severity="warning"
+                    title={t("calendar_error")}
+                    message={item.error?.message}
+                    actions={
+                      <div className="w-32">
+                        <DisconnectIntegration
+                          credentialId={item.credentialId}
+                          trashIcon
+                          onSuccess={props.onChanged}
+                          buttonProps={{ className: "border border-gray-300" }}
```

Comment:
```suggestion
                          buttonProps={{ size: "icon", color: "secondary" }}
```

Reply:
Not needed, but good catch if I hadn't tweaked the `DisconnectIntegration` component 👍